scripts/build.py

```diff
@@ -426,11 +429,8 @@ def process_html(source_file_stream, target_file_stream, file_hashes):
         # This is because html paths are used by backend and we work with
         # paths without hash part in backend.
         if not filepath.endswith('.html'):
-            # The path to the required file in the compiled HTML is in
-            # url-style, while on Windows, the path here will be in
-            # Windows-Style. We need to convert the Windows-Style path to url
-            # style so the target path can be successfully replaced with the
-            # one with hash.
+            # The pathes to the required files in the compiled HTML is in
```

Comment:
pathes --> paths

Also, in this case -- what is imposing this requirement? It's not clear from the docstring. (I understand the others: one is because java cmdline requires it and the others are because of hashes.json, but I don't get, from the comment, what's imposing this requirement for this variable.)

Reply:
Sorry, I am not sure how to answer this question. From my understanding, developers wrote the path in URL style. Thus, when on Windows, the path we are searching should also be converted to URL style so it can be successfully replaced to the one with hash inserted. I don't know in that case what is imposing the requirement.

Reply:
Oh, I see. I think what you want is:

"This reconstructs the hashed version of the URL, so that it can be used to substitute the raw URL in the HTML file stream."

Reply:
Updated